### ISSUE TITLE
Ability to use Multiple IP Addresses (Do not bind to All interfaces)

### ISSUE THREAD

**Issue by JRodriguez556** [kgretzky/evilginx2#522]
Currently, evilginx2 binds to ::::80,443,53. 

Requesting ability to specify bind IP when starting evilginx2 so that multiple domains can be used with multiple IP Addresses.

(This should also automatically set the "config ip x.x.x.x" to the IP specified on launch)

**Comment by CraftedCat** [comment]
You can run evilginx with custon config file ./evilginx2 -c

**Comment by JRodriguez556** [comment]
> You can run evilginx with custon config file ./evilginx2 -c

Yes, you can set the IP, but the Evilginx process will bind to all interfaces.

**Comment by kgretzky** [comment]
This may probably be useful to have an option to bind to specific interface. Noted.
